### PR TITLE
Attempt 2 - Xiaomi Redmi 5 (Rosy) inital support

### PR DESCRIPTION
**Pull request by hiprivsid**
- Added support for rosy and tested it

### CODE REVIEW

**Comment by hiprivsid** [comment]
@barni2000 you can handover this pr

**Comment by hiprivsid** [comment]
@barni2000 okay, i will add panel selection

**Comment by barni2000** [comment]
```
panel {
        compatible = "xiaomi,rosy-panel";

        qcom,mdss_dsi_ft8613_ebbg_5p7_720p_video {
                compatible = "xiaomi,rosy-ft8613-ebbg";
        };

        qcom,mdss_dsi_ft8613_csot_5p7_720p_video {
                compatible = "xiaomi,rosy-ft8613-csot";
        };

        qcom,mdss_dsi_ft8006m_boe_5p7_720p_video {
                compatible = "xiaomi,rosy-ft8006m-boe";
        };
};
```

**Comment by hiprivsid** [comment]
now panel selection added

**Review by barni2000 — CHANGES_REQUESTED** [review]
Please use tabs for indentation, device trees are indented with tabs.

File: dts/sdm450-xiaomi-rosy.dts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+/dts-v1/;
+
+/include/ "msm8953.dtsi"
+
+/ {
+        qcom,board-id = <0x1000b 1>;
+        qcom,msm-id = <338 0x0>;
+
+        model = "Xiaomi Redmi 5 (rosy)";
+        compatible = "xiaomi,rosy", "qcom,msm8953", "lk2nd,device";
```
Comment (barni2000) [inline dts/sdm450-xiaomi-rosy.dts:12]: here

**Comment by barni2000** [comment]
Please squash your commits together

**Comment by hiprivsid** [comment]
> Please squash your commits together

in the past someone wrote to me about this, but I still don't understand how

**Comment by barni2000** [comment]
> > Please squash your commits together
> 
> in the past someone wrote to me about this, but I still don't understand how
Make a useful commit message also, check the other device commits.
https://www.git-tower.com/learn/git/faq/git-squash/

**Comment by hiprivsid** [comment]
squashed

**Pull request merged**